src/tests/test_utils/common_test_utils/src/test_common.cpp

```diff
@@ -30,10 +30,12 @@ TestsCommon::TestsCommon()
     : PGLink(new utils::PostgreSQLLink(this))
 #endif
 {
+#ifndef __APPLE__  // TODO: add getVmSizeInKB() for Apple platform   
```

Comment:
@ilyachur I decided to remove use `getVmSizeInKB()` from here in order to throw clear exception in any other use (some other test which will rely on it's implementation)